src/components/map/CuriosityMap.tsx

```diff
@@ -1,9 +1,26 @@
-import React from "react";
+import React, {useState} from "react";
+import './Map.scss'
 
 export const CuriosityMap: React.FunctionComponent = () => {
+  const [date, setDate] = useState('0')
+
+  function handleClick(day:string) {
```

Comment:
Prefer arrow function definition:
```suggestion
  const handleClick = (day:string) => {
```